README.md

```diff
@@ -121,7 +127,8 @@ This tool comes with some inputs that allow users to override the default behavi
 | Backport Branch Names       | --bp-branch-name        | N            | Comma separated lists of the backporting pull request branch names, if they exceeds 250 chars they will be truncated                                                           | bp-{target-branch}-{sha1}...{shaN}       |
 | Labels       | --labels        | N            | Provide custom labels to be added to the backporting pull request                                                           | []       |
 | Inherit labels       | --inherit-labels        | N            | If enabled inherit lables from the original pull request                                                           | false       |
-| No squash       | --no-squash        | N            | If provided the backporting will try to backport all pull request commits without squashing                                                           | false       |
+| No squash       | --no-squash        | N            | Backport all commits found in the pull request. The default behavior is to only backport the first commit that was merged in the base branch. | |
+| No squash auto  | --auto-no-squash   | N            | If the pull request was merged or is open, backport all commits. If the pull request commits were squashed, backport the squashed commit. | |
```

Comment:
I'd change to `Auto no squash` otherwise it could be confusing as people could think that setting it means to disable some sort of behavior
```suggestion
| Auto no squash  | --auto-no-squash   | N            | If the pull request was merged or is open, backport all commits. If the pull request commits were squashed, backport the squashed commit. | |
```